lib/ansible/modules/system/ufw.py

```diff
@@ -461,7 +461,7 @@ def ufw_version():
                 execute(cmd + [[command], [value]])
 
         elif command == 'default':
-            if params['direction'] not in ['outgoing', 'incoming', 'routed']:
+            if params['direction'] not in ['outgoing', 'incoming', 'routed', None]:
                 module.fail_json(msg='For default, direction must be one of "outgoing", "incoming" and "routed".')
```

Comment:
Maybee you should change the message too. 

'For default, direction must be empty or one of "outgoing", "incoming" and "routed"" '

Reply:
I've changed both messages.